src/app/(home)/activate/page.tsx

```diff
@@ -56,8 +59,7 @@ const Activate = () => {
 			try {
 				const isLocalhost =
 					window.location.hostname.includes(mockLocalhostStr) || window.location.hostname.includes(mockLocalhostUrl);
-
-				if (baseUrl && !isLocalhost && uid && token) {
+				if (baseUrl && !isLocalhost && uid && token && !isActivationPage) {
```

Comment:
if(!isActivationPage)
can you move this condition in src/components/mainLayout/mainHeader/mainheader.tsx in useEffect hook with sending validateToken request?